mne/viz/misc.py

```diff
@@ -569,3 +569,104 @@ def plot_dipole_amplitudes(dipoles, colors=None, show=True):
     if show:
         fig.show(warn=False)
     return fig
+
+
+def plot_tfr_gfp(tfr, baseline=None, mode='mean', colors='auto',
+                 summarize=False, title=None, show=True):
+    """ Show time and frequency resolved global field power
```

Comment:
`power.`

Maybe also indicate that this corresponds to RMS, since some users refer to it accordingly